build.cake

```diff
@@ -361,15 +362,77 @@ Task("binderate-fix")
                 if ( ! DirectoryExists(dir_group) )
                 {
                     Warning($"  		Creating {dir_group}");
-                    //CreateDirectory(dir_group);
+                    CreateDirectory(dir_group);
                 }
+
                 string dir_artifact = $"{dir_group}/{artifactId}";
-                if ( ! DirectoryExists(dir_group) )
+                if ( ! DirectoryExists(dir_artifact) )
                 {
                     Warning($"     			Creating artifact folder : {dir_artifact}");
-                    //CreateDirectory(dir_group);
+                    CreateDirectory(dir_artifact);
+                    CreateDirectory($"{dir_artifact}/Transforms/");
+                    CreateDirectory($"{dir_artifact}/Additions/");
                 }
+				else
+				{
+					continue;
+				}
 
+				if ( ! FileExists($"{dir_artifact}/Transforms/Metadata.xml"))
+				{
+					Warning($"     				Creating file : {dir_artifact}/Metadata.xml");
+					CopyFile
+					(
+						$"./source/template-group-id/template-artifact/Transforms/Metadata.xml",
+						$"{dir_artifact}/Transforms/Metadata.xml"
+					);
+				}
```

Comment:
Is there a reason to create these files?

Reply:
When you add new artifacts in `config.json` it will create fodler structure in `./source/groupId/artifactId/` so user don't have to add those manually.

Reply:
Ah, that is a nice thing.